libs/perun/components/src/lib/perun-header/perun-header.component.html

```diff
@@ -0,0 +1,67 @@
+<mat-toolbar
+  [ngStyle]="{'background': bgColor}"
+  class="sticky-top header-container mat-elevation-z3">
+  <div id="nav-menu-icons">
+    <button
+      (click)="onToggleSidenav()"
+      *ngIf="showToggle"
+      [ngStyle]="{'color': iconColor}"
+      aria-label="Menu"
+      mat-icon-button>
+      <mat-icon>menu</mat-icon>
+    </button>
+
+    <a
+      [innerHTML]="logo"
+      [routerLink]="disableLogo ? [] : ['/']"
+      class="logo-container mt-auto mb-auto"
+      queryParamsHandling="merge"></a>
+
+    <p [ngStyle]="{'color': textColor}" class="mt-auto mb-auto hide-label page-subtitle">
+      {{label}}
+    </p>
+  </div>
+
+  <div *ngIf="isDevel" id="nav-devel-info">
+    <span class="text-warning"> ~ 🗲&nbsp; </span>
+    <span class="text-danger"> DEVEL </span>
+    <span class="text-warning"> 🗲 ~ </span>
+  </div>
```

Comment:
I am not 100 % sure, but since we don't have this label on our devel, this div is maybe no longer necessary. If it is true, then you can delete also the property in ts file, the property is_devel in ConfigProperties.ts, and also in all default configs.
```suggestion
```